build.sbt

```diff
@@ -21,16 +20,18 @@ resolvers ++= Seq(
 
 libraryDependencies ++= Seq(
   "org.webjars" % "swagger-ui" % "4.1.3",
-  "org.broadinstitute.dsde" %%  "rawls-model" % rawlsModelV
-    exclude("bio.terra", "workspace-manager-client")
-    exclude("com.google.api-client", "google-api-client"),
+  "com.typesafe.scala-logging" %% "scala-logging" % "3.9.5",
+  "com.typesafe.akka" %% "akka-http-spray-json" % "10.2.9",
```

Comment:
adding these back in as direct dependencies because they were coming from rawls-model